CHANGELOG.md

```diff
@@ -23,6 +23,7 @@ ENHANCEMENTS:
 BUG FIXES:
 
 * core: Sensitive values will now be tracked more accurately in state and plans, preventing unexpected updates with no apparent changes [GH-34567]
+* CLI-driven runs were colorizing strings returned as error messages when a policy soft fails. The fix removes this call to colorize in #34473.
```

Comment:
A suggestion on the wording:
```
Removes colorized strings from error messages when a policy soft fails #34473 
```

Reply:
The change is happening in `backend/remote` and `cloud`

Reply:
The language sounds like there is part of an error message that is colorized, and that part is being removed, instead of the entire string returned as an error message is just no longer colorized. I'll build from your suggestion.